### PR TITLE
Add FCN and d_tau to MSG_EPHEMERIS_GLO

### PR DESCRIPTION
**Pull request by dt-exafore**
Based on https://github.com/swift-nav/exafore_planning/issues/750

### CODE REVIEW

File: haskell/sbp.cabal [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name:                  sbp
-version:               2.2.3
+version:               2.2.3-dirty
```
Comment (peddie) [inline haskell/sbp.cabal]: Shouldn't this become `2.2.4`?

Comment (dt-exafore) [inline haskell/sbp.cabal]: I don't know. It was autogenerated.

Comment (mfine) [inline haskell/sbp.cabal]: @dt-exafore it means when you generated the code, there were uncommitted changes - this should get backed out.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "libsbp",
-  "version": "2.2.3",
+  "version": "2.2.3-dirty",
```
Comment (mfine) [inline package.json]: @dt-exafore also this.

File: haskell/sbp.cabal (hunk 1 above)
Comment (dt-exafore) [inline haskell/sbp.cabal]: @mfine Does it mean that the process is, first, commit my changes in .yaml and python tests if any, second, generate other code and finally make one more commit with generated files?
I usually make change, generate code and commit everything at once.

Comment (mfine) [inline haskell/sbp.cabal]: Process is usually commit changes in `.yaml`, generate code, fix tests if any and commit. The `-dirty` means you ran the generator with uncommitted changes:

```
➜  ~/repos/libsbp git:(master) git describe --tags --dirty --always
v2.2.3
➜  ~/repos/libsbp git:(master) echo woops > README.md 
➜  ~/repos/libsbp git:(master) ✗ git describe --tags --dirty --always
v2.2.3-dirty
➜  ~/repos/libsbp git:(master) ✗ 
```

**Comment by dt-exafore** [comment]
@mfine I generated it again and it removes "-dirty", however the version is still 2.2.3, should it be 2.2.4?

**Comment by mfine** [comment]
@dt-exafore no, that usually happens during the releasing of the library.

**Comment by nsirola** [comment]
Before merging, let's check if there were any more GLO ephemeris fields that we could add in the same go.

**Comment by nsirola** [comment]
GLO ephemeris / non-immediate data elements that are not yet being reported in the SBP message (ICD section 4.4):

P - technological parameter (not relevant?)
P1 - flag of the immediate data updating (not relevant, already included in fit interval?)
P2 - flag of oddness or evenness of the value t_b (not relevant??)
P3 - flag indicating a number of satellites for which almanac is 
transmitted within given frame (not relevant?)
P4 - flag to show that ephemeris parameters are present (not relevant?)


Additionally there's the GLONASS almanac contains the Keplerian orbit elements etc. that is not decoded yet, save for:

tau_c - GLONASS time scale correction to UTC(SU) time
tau_gps - correction to GPS time relative to GLONASS time

**Comment by anth-cole** [comment]
lgtm - I don't think the others are needed in the ephemeris

**Pull request merged**